src/main/java/com/conveyal/datatools/manager/auth/Auth0Connection.java

```diff
@@ -138,7 +145,7 @@ private static void remapTokenValues(Map<String, Object> jwt) {
      * tables in the database.
      */
     public static void checkEditPrivileges(Request request) {
-        if (authDisabled() || inTestingEnvironment()) {
+        if (getDefaultAuthDisabled() || inTestingEnvironment()) {
```

Comment:
Everywhere that previously checked `authDisabled()` should check `Auth0Connection#authDisabled` or some new method `Auth0Connection#isAuthDisabled`

Reply:
Updated.